go/vt/sqlparser/parse_test.go

```diff
@@ -2457,6 +2457,34 @@ func TestValid(t *testing.T) {
 	}
 }
 
+func TestParsingColumnsStartingWithNumbers(t *testing.T) {
+	tests := []parseTest{
+		{
+			input:  "create table t (1a int)",
```

Comment:
Need tests of these identifiers in other statement types. Make sure you cover insert, update, delete, and a couple DDL statements

Reply:
Add some tests for table names with the same semantics

Also db names, and qualified table names, e.g.

`CREATE DATABASE 1A`
`INSERT INTO 1A.2B VALUES ....`